### PR TITLE
Add examples and documentation for the Persona component.

### PR DESCRIPTION
**Pull request by GillesTourreau**
# Pull Request

## 📖 Description

Add examples and documentation for the `Persona` component.

## 👩‍💻 Reviewer Notes

- I did not make changes to the component, I just add example in the demo project.
- I added a `Persona` item in the navigation menu.
- Do not hesitate to fix my english (I am native french guy...  :laughing:).

### General

- [X] I have added tests for my changes.
- [X] I have tested my changes.
- [X] I have updated the project documentation to reflect my changes.
- [X] I have read the [CONTRIBUTING](https://github.com/Microsoft/fluentui-blazor/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

**Comment by GillesTourreau** [comment]
@microsoft-github-policy-service agree

**Comment by dvoituron** [comment]
Bonjour Gilles,

Why do you want to add these samples? We already have a sample and doc on the [Autocomplete](https://www.fluentui-blazor.net/Autocomplete) page. Is it not enough ?

File: examples/Demo/Shared/Pages/Persona/Examples/PersonaDefault.razor [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿<FluentPersona Name="Kevin Sturgis"
+               Status="PresenceStatus.Available"
+               StatusSize="PresenceBadgeSize.Small"
+               StatusTitle="He is available"
+               Image="https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png"
```
Comment (dvoituron) [inline examples/Demo/Shared/Pages/Persona/Examples/PersonaDefault.razor:5]: Could you use the `@DataSource.SamplePicture` image? To be independent of external resources (which can be removed later)

File: examples/Demo/Shared/Pages/Persona/Examples/PersonaDismiss.razor [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<div hidden="@this.Hidden">
+    <FluentPersona Name="Kevin Sturgis"
+                   Status="PresenceStatus.Available"
+                   StatusSize="PresenceBadgeSize.Small"
+                   StatusTitle="He is available"
+                   Image="https://res-1.cdn.office.net/files/fabric-cdn-prod_20230815.002/office-ui-fabric-react-assets/persona-male.png"
```
Comment (dvoituron) [inline examples/Demo/Shared/Pages/Persona/Examples/PersonaDismiss.razor:6]: Same... Could you use the `@DataSource.SamplePicture` image? To be independent of external resources (which can be removed later)

**Comment by GillesTourreau** [comment]
> Bonjour Gilles,
> 
> Why do you want to add these samples? We already have a sample and doc on the [Autocomplete](https://www.fluentui-blazor.net/Autocomplete) page. Is it not enough ?

Hello Denis,

I didn't remark that `Persona` component was included in the `Autocomplete` demo page. My co-workers and me, we discovered this component in the demo website for React with a simple search on Google... And after investigation we discovered this component by searching the class in the NuGet package...

I really like the [Fluent UI Blazor components library](https://www.fluentui-blazor.net/) website, because it contains a good documentation and demo for each components. I find it a shame that some components are hidden (mixed) with other components and not visible in the navigation bar on the left. The navigation bar on the left, should reflect all the components available in the Blazor library to show to the developers, all the features of this library...

File: examples/Demo/Shared/Pages/Persona/Examples/PersonaDismiss.razor (hunk 2 above)
Comment (GillesTourreau) [reply examples/Demo/Shared/Pages/Persona/Examples/PersonaDismiss.razor]: I fixed both remarks.

**Comment by dvoituron** [comment]
Yes, you're probably right. We could add these example pages for greater clarity.

**Review by dvoituron — APPROVED** [review]

**Pull request merged**